src/main/kotlin/no/skatteetaten/aurora/boober/service/vault/VaultService.kt

```diff
@@ -171,8 +172,9 @@ class VaultService(
 
         val vaults = findAllVaultsInVaultCollection(vaultCollectionName)
         vaults.forEach { vault: EncryptedFileVault ->
+            val newEncryptorWrapper = EncryptorWrapper(encryptionService.encryptor.key)
```

Comment:
Her må du vel bruke newKey som verdi til EncryptorWrapper?

Reply:
jeg har fått oppdatert og lagt til en test for reEncrypt av vault